.codeclimate.yml

```diff
@@ -0,0 +1,36 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
```

Comment:
I think all of those extensions get added because they're all potentially analyzeable by the duplication engine. We should make that smarter ;)